setup.py

```diff
@@ -109,6 +109,9 @@ def get_extras_require() -> Dict[str, List[str]]:
             "dask-ml",
             "botorch ; python_version>'3.6'",
             "fastai",
+            "jax",
+            "optax",
+            "dm-haiku @ git+https://github.com/deepmind/dm-haiku",
```

Comment:
This is because I follow [the haiku's documentation](https://github.com/deepmind/dm-haiku#installation). I'm not sure `pip install dm-haiku` is an official way to install, but [the pip page](https://pypi.org/project/dm-haiku/) is supposed to be maintained by deepmind workers...

Reply:
Hmm, as you said, `ml-haiku` on PyPI seems to be maintained by deepmind. So I think we can use `dm-haiku` on PyPI, though the documentation recommended to install the software from source. What do you think about it?

Reply:
Either is fine for me, but I think only `pip install dm-haiku` is simpler to maintain for optuna developers. I will update this part. Thanks!